mantis-server/mantis-server-agent/src/main/java/io/mantisrx/server/agent/metrics/cgroups/CpuAcctsSubsystemProcess.java

```diff
@@ -61,4 +70,20 @@ public void getUsage(Usage.UsageBuilder resourceUsageBuilder) throws IOException
             log.warn("quota={} & period={} are not configured correctly", quota, period);
         }
     }
+
+    private void handleV2(Usage.UsageBuilder resourceUsageBuilder) throws IOException {
+        Map<String, Long> cpuStats = cgroup.getStats(null, "cpu.stat");
+        resourceUsageBuilder
+            .cpusUserTimeSecs(cpuStats.getOrDefault("user_usec", 0L) / 1000.0)
+            .cpusSystemTimeSecs(cpuStats.getOrDefault("system_usec", 0L) / 1000.0);
+
+        List<Long> metrics = cgroup.getMetrics(null, "cpu.max");
+        if (metrics.size() != 2) {
+            log.warn("cpu.max metrics={} are not configured correctly", metrics);
+        } else {
+            Long quota = metrics.get(0);
```

Comment:
The value of this file is a literal string `max` when there is no quota. I can't tell what the `long quota` is going to be in that case.

Reply:
In that case, how do you think one can evaluate the number of cpus allocated to the container?

Reply:
In that case (`max`) then the cgroup isn't allocating anything to the container, per-se. At that point I suppose one would could look at all the cpus on the system, but that would be an integer like `16` and not of the same unit as these in the cpu.stat.

Reply:
I say we not worry about it unless someone complains.